### PR TITLE
FISH-9156: update of weld versions and dependencies

### PR DESCRIPTION
**Pull request by breakponchito**
Updating to the latest version of weld and payara arquillian to execute cdi lang tck

Manual testing

execution of cdi lang without any issue:

image

Environment:
ubuntu 20.04, azul jdk 21, maven 3.8.6

### CODE REVIEW

**Comment by breakponchito** [comment]
> Will conflict with #65

I already merged your changes and also the build is passing:
https://jenkins.payara.fish/view/Development%20Testing/job/JakartaEE-11-TCK-Development-FISH-9156/4/

`
+ java -jar target/tck-summarizer-1.0-SNAPSHOT.jar ./samples/*.txt testSet cdi-langmodel
Parsing test set results
### cdi-langmodel
 
Completed running 1 tests
Number of tests failed 0
Number of tests with errors 0
Number of tests skipped 0
`

File: cdi-langmodel-tck/pom.xml [hunk 1]
```diff
@@ -19,9 +19,9 @@
         <maven.compiler.release>21</maven.compiler.release>
         <cdi.tck.version>4.1.0</cdi.tck.version>
         <cdi.api.version>4.1.0</cdi.api.version>
-        <weld.version>6.0.0.Beta1</weld.version>
+        <weld.version>6.0.0.Beta4</weld.version>
         <arquillian.version>1.8.0.Final</arquillian.version>
```
Comment (Pandrex247) [inline cdi-langmodel-tck/pom.xml:23]: Redundant property - defined in parent pom
```suggestion
```

File: cdi-langmodel-tck/pom.xml [hunk 2]
```diff
@@ -19,9 +19,9 @@
...
-        <payara.arquillian.version>3.0</payara.arquillian.version>
+        <payara.arquillian.version>4.0.alpha1</payara.arquillian.version>
```
Comment (Pandrex247) [inline cdi-langmodel-tck/pom.xml:24]: Redundant property - defined in parent pom
```suggestion
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

File: cdi-langmodel-tck/pom.xml (hunk 2 above)
Comment (breakponchito) [reply cdi-langmodel-tck/pom.xml]: already removed

File: cdi-langmodel-tck/pom.xml (hunk 1 above)
Comment (breakponchito) [reply cdi-langmodel-tck/pom.xml]: already removed

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**